### PR TITLE
[TOOLS-1733]Add fields role and globally visible

### PR DESCRIPTION
**Pull request by silvia-shimabuko**

### CODE REVIEW

File: src/main/java/objective/taskboard/team/TeamsController.java [hunk 1]
```diff
@@ -113,34 +116,44 @@ public TeamsController(
...
         membersToAdd.forEach(userTeam -> team.getMembers().add(userTeam));
 
+        List<MemberDto> membersToUpdate = data.members.stream()
+            .filter(member -> team.getMembers().stream().noneMatch(userTeam -> {
+                return userTeam.getUserName().equals(member.name) && userTeam.getRole() == member.role; }))
+            .collect(toList());
+
+        membersToUpdate.forEach(tela -> {
+            UserTeam userTeam = team.getMembers().stream().filter(bd -> bd.getUserName().equals(tela.name)).findFirst().get();
+            userTeam.setRole(tela.role);;
```
Comment (objective-solutions-user) [inline src/main/java/objective/taskboard/team/TeamsController.java]: MINOR Remove this empty statement. [rule](http://10.44.1.250:9000/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)

**Comment by objective-solutions-user** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: web-app/src/app/teams/teams/team/team.component.ts [hunk 2]
```diff
@@ -86,8 +91,7 @@ export class TeamComponent extends ComponentLeaveConfirmation implements OnInit
     }
 
     removeMember(userName: string) {
-        const index = this.team.members.indexOf(userName);
-        this.team.members.splice(index, 1);
+        this.team.members = this.team.members.filter(member => member.name !== userName);
```
Comment (jhonypg) [inline web-app/src/app/teams/teams/team/team.component.ts]: Ver se consegue manter no mesmo padrão.

File: web-app/src/app/teams/teams/teams.component.html [hunk 3]
```diff
@@ -35,7 +35,7 @@ <h2 id="tb-page-title" class="obj-title-large title">
                     </td>
                     <td>
                         <span class="ellipsis">
-                            {{ team.members.join(', ') }}
+                            {{ getListMembersName(team.members).join(', ') }}
```
Comment (jhonypg) [inline web-app/src/app/teams/teams/teams.component.html]: Colocar join dentro do método.

**Pull request closed without merging**